portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -25,12 +25,16 @@
   private String link;
   private String description;
   private String id;
+  private String userEmail;
+  private String displayedName;
 
-  public Comment(String location, String link, @Nullable String description, String id) {
+  public Comment(String location, String link, @Nullable String description, String id, String userEmail, String displayedName) {
```

Comment:
ok for now, but if this keeps getting longer, refer to https://drive.google.com/file/d/1PJcAtx0N3fLJdJ9lLxeF5nALGuMxxy_7/view